### DISCUSSION TITLE
Vulkan device doesn't support required feature: shaderDrawParameters

### DISCUSSION THREAD

**Discussion by TacoGuyAT** [skyline-emu/skyline#1717] [Q&A] [answered]
I'm unsure if my phone (Meizu 16X, Snapdragon 710, Adreno 616) actually supports this extension, because the only way I found for now is "Vulkan Hardware Capability Viewer" tool from Google Play, which says "r.1" in VK_KHR_shader_draw_parameters value, but I suppose it's a must-have one for drawing with Vulkan. 
```
E15EmuMainAn uncaught exception has occurred: Vulkan device doesn't support required feature: shaderDrawParameters
I15EmuMainEmulation has ended in 15ms
```
![image](https://user-images.githubusercontent.com/35270005/180690911-9453df80-576c-4b1e-8c0e-1a1d35ffb25d.png)

**Comment by yhak22** [comment]
Low end phone .prob

**Comment by PixelyIon** [comment, marked as answer]
The Adreno 616 is a bit too low end if it only supports that as an extension and not as a feature, we will not be adding support for it unfortunately.

**Reply by TacoGuyAT** [reply]
Ah, ok, thanks for the answer

**Comment by ghost** [comment]
710 is old but still better or at least on par with the newly released 695 and 680
